### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by ohadschn**
Fixed some typos and grammar errors.
Added some C# samples.

**I've signed the LCA**

### CODE REVIEW

**Comment by ddavison** [comment]
This is fantastic!  Thanks so much for contributing this!

Since i'm not .NET, i'll leave this judgement to somebody else, but LGTM 👍 

The only reason I won't commit this in personally, is that I can't say for sure that [this piece](https://github.com/SeleniumHQ/www.seleniumhq.org/pull/78/files#diff-81e71c481c0b99a2867f6a4877e6800aR4) is entirely accurate.  I'm not sure if using NuGet is the recommended solution.

**Comment by lukeis** [comment]
awesome, thanks for the contribution!

**Comment by ohadschn** [comment]
Glad you liked it guys :)

NuGet is generally the preferred method for dependency management in .NET, don't see any reason why Selenium would be different... And so far it works very well out of the box for me :+1:

File: src/main/rst/03_webdriver.rst [hunk 1]
```diff
@@ -270,9 +269,9 @@ Locating UI Elements (WebElements)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Locating elements in WebDriver can be done on the WebDriver instance itself or on a WebElement.
-Each of the language bindings expose a "Find Element" and "Find Elements" method. The first returns
-a WebElement object otherwise it throws an exception. The latter returns a list of WebElements, it can
-return an empty list if no DOM elements match the query.
+Each of the language bindings exposes a "Find Element" and "Find Elements" method. The former returns
```
Comment (bryan-gordon4) [inline src/main/rst/03_webdriver.rst]: this needs to be "expose"

Comment (jimevans) [inline src/main/rst/03_webdriver.rst]: Nope, "exposes" is grammatically correct. "Each ... exposes"

**Pull request merged**